opg-phpunit-1604/Dockerfile

```diff
@@ -1,22 +1,20 @@
 FROM registry.service.opg.digital/opg-php-fpm-1604
 
-ARG PHPUNIT_VERSION="6.4"
-
 RUN cd /tmp && \
-    wget https://phar.phpunit.de/phpunit-${PHPUNIT_VERSION}.phar && \
-    chmod +x phpunit-${PHPUNIT_VERSION}.phar && \
-    mv phpunit-${PHPUNIT_VERSION}.phar /usr/local/bin/phpunit
-
-# enable php-xdebug for code coverage
-RUN cp /etc/php/7.0/xdebug-enable.ini /etc/php/7.0/fpm/conf.d/20-xdebug.ini && \
-    cp /etc/php/7.0/xdebug-enable.ini /etc/php/7.0/cli/conf.d/20-xdebug.ini
-
-# We need version 1.2 of the mongo extension
-RUN apt-get remove -y php-mongodb
+    wget https://phar.phpunit.de/phpunit.phar && \
```

Comment:
We prefer to pin versions of tools like this, if there is a new version, rather update the var. Some versions of phpunit introduce non bc changes